src/main/java/uk/gov/pay/webhooks/webhook/resource/WebhookResponse.java

```diff
@@ -12,13 +14,20 @@
 import java.util.List;
 
 public record WebhookResponse(
+        @Schema(example = "eo29upsdkjlk3jpwjj2dfn12")
         @JsonProperty("service_id") String serviceId,
         @JsonProperty("live") Boolean live,
+        @Schema(example = "https://example.com")
         @JsonProperty(FIELD_CALLBACK_URL) String callbackUrl,
+        @Schema(example = "Webhook description")
         @JsonProperty(FIELD_DESCRIPTION) String description,
+        @Schema(example = "gh0d0923jpsjdf0923jojlsfgkw3seg")
```

Comment:
Can we maybe do a more friendly example here? e.g. `some description` or `my awesome webhook`